components/collective-page/sections/Transactions.js

```diff
@@ -112,9 +111,7 @@ const SectionTransactions = props => {
           {loading ? (
             <LoadingPlaceholder height={600} borderRadius={8} />
           ) : (
-            <GraphQLContext.Provider value={transactionsQueryResult}>
```

Comment:
As per @kewitz's feedback, I removed all these since there was no consumer for them (thus it seems to be unused)